sdk/python/feast/infra/contrib/spark_kafka_processor.py

```diff
@@ -116,9 +117,27 @@ def __init__(
         # data_source type has been checked to be an instance of KafkaSource.
         self.data_source: KafkaSource = self.data_source  # type: ignore
 
+    def _create_infra_if_necessary(self):
+        if self.fs.config.online_store is not None and getattr(
+            self.fs.config.online_store, "lazy_table_creation", False
+        ):
+            print(
+                f"Online store {self.fs.config.online_store.__class__.__name__} supports lazy table creation and it is enabled"
+            )
```

Comment:
Should we use print or logger?

Reply:
Currently print statement inside the spark tasks. Loggers need to be configured but it's not the case currently.